src/service/overwritePersonaAttribute.ts

```diff
@@ -1,10 +1,9 @@
+// tslint:disable-next-line:no-unused
 import OverwritePersonaAttributeOptions from // tslint:disable-line:import-spacing
   '../serviceFactory/options/OverwritePersonaAttributeOptions';
+// tslint:disable-next-line:no-unused
```

Comment:
Like I said, let's get rid of the options and result interfaces in the service since they're not needed.